<?xml version="1.0"?>

<launch>
    <rosparam file="$(find stair_detection)/config/stair_config.yaml" command="load" /> 

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find stair_detection)/config/offline_opt_visual.rviz" />

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />

<!--     <node pkg="dynamic_reconfigure" type="dynparam" name="dyn_loader" args="load /offline_opt $(find stair_detection)/config/stair_config_last.yaml"/>-->

    <node name="tf_world_set" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 1 world odom 10" /> 

    <node pkg="stair_detection" type="visualization_node" name="visualizer"  />

    <node pkg="stair_detection" type="offline_opt_node" name="offline_opt" output="screen" launch-prefix="gnome-terminal -e"/>    

</launch>
